src/herder/HerderSCPDriver.cpp

```diff
@@ -1227,6 +1237,32 @@ HerderSCPDriver::wrapStellarValue(StellarValue const& sv)
     return res;
 }
 
+void
+HerderSCPDriver::cacheTxSetValidity(ApplicableTxSetFrame const& txSet,
```

Comment:
nit: This name is potentially misleading - the way I've interpreted this first time is that this caches whether tx set is valid or not, while in it unconditionally records that tx set is valid. Should this be named `cacheValidTxSet` for clarity?